pages/contact.js

```diff
@@ -0,0 +1,66 @@
+import React, {Component} from 'react';
```

Comment:
supernit: space after {

```suggestion
import React, { Component } from 'react';
```

Reply:
if we make this component a functional one, we wont need to import `React, { Component } from 'react'` anymore cause I dont think it is being used at all 🥇 YAY less code!